hera_sim/visibilities/healvis_wrapper.py

```diff
@@ -59,9 +78,11 @@ def sky_model(self):
     @cached_property
     def observatory(self):
         """A healvis :class:`healvis.observatory.Observatory` instance"""
-        return setup_observatory_from_uvdata(
```

Comment:
I think just returning it is fine -- no need for an extra variable

Reply:
Done.